LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/JsonHandlers/LnsData.cs

```diff
@@ -3,25 +3,19 @@
 
 namespace LoRaWan.NetworkServer.BasicsStation.JsonHandlers
 {
+    using System;
     using System.Linq;
     using System.Text.Json;
 
     public static class LnsData
     {
         internal static readonly IJsonReader<LnsMessageType> MessageTypeReader =
```

Comment:
Seems excessive to have this here when it's only used in `LnsProtocolMessageProcessor`. How about moving it as a private field of that type so it's closest to its use?

Reply:
I would keep everything "Json" related in this LnsData JsonHandler, to have it logically isolated here. If you have a strong opinion on this, I can fix it 👍